ecommerce/management/commands/invalidate_payment_coupons.py

```diff
@@ -0,0 +1,79 @@
+"""
+Disables coupons in a batch, either by parsing a CSV file or by iterating
+through a coupon payment. 
+"""
+
+from django.core.management import BaseCommand, CommandError
+
+from ecommerce.models import CouponPayment, Coupon
+
+
+class Command(BaseCommand):
+    """
+    Disables coupons in the system.
+    """
+
+    help = "Disables coupons in the system."
+
+    def add_arguments(self, parser) -> None:
+        parser.add_argument(
+            "--payment",
+            "-p",
+            nargs="?",
+            type=str,
+            help="The payment to iterate through.",
+            dest="payment",
+        )
+
+        parser.add_argument(
+            "--file",
+            "-f",
+            nargs="?",
+            type=str,
+            help="File of codes to process.",
+            dest="codefile",
+        )
+
+    def handle(self, *args, **kwargs):
+        if (
+            "payment" not in kwargs
+            or "codefile" not in kwargs
+            or kwargs["payment"] == ""
+            or kwargs["codefile"] == ""
+        ):
+            raise CommandError(
+                "Please specify a payment to deactivate or a code file to process."
+            )
```

Comment:
Currently running the command without any params goes to the `else` block trying to fetch the `None` file.

I think this can also be simplified as and this would also work properly without any arguments:
```suggestion
        if not (
            kwargs["payment"]
            or kwargs["codefile"]
        ):
            raise CommandError(
                "Please specify a payment to deactivate or a code file to process."
            )
```